starboard/shared/posix/page_internal.cc

```diff
@@ -92,6 +96,22 @@ void* SbPageMap(size_t size_bytes, int flags, const char* unused_name) {
   return ret;
 }
 
+bool SbPageUnmap(void* ptr, size_t size_bytes) {
+  SbAtomicNoBarrier_Increment(&s_tracked_page_count, -GetPageCount(size_bytes));
+  return SbPageUnmapUntracked(ptr, size_bytes);
+}
+
+bool SbPageProtect(void* virtual_address, int64_t size_bytes, int flags) {
+  int mmap_protect = SbMemoryMapFlagsToMmapProtect(flags);
+  return mprotect(virtual_address, size_bytes, mmap_protect) == 0;
+}
+
+size_t SbPageGetMappedBytes() {
+  return static_cast<size_t>(SbAtomicNoBarrier_Load(&s_tracked_page_count) *
+                             kSbMemoryPageSize);
+}
+#endif  // SB_API_VERSION < 16
+
 void* SbPageMapFile(void* addr,
```

Comment:
Is this not being deprecated because it's used in starboard extensions and we're keeping those in SB16?

Reply:
It is used by the Starboard extension. Once we migrate the SbFile I can try to eliminate the extension too, but for now it is too hard as it need a file descriptor.